resolvers/deno/npm/mod.rs

```diff
@@ -489,3 +538,45 @@ impl<
     }
   }
 }
+
+pub fn definitely_typed_package_name(name: &str) -> String {
+  format!("@types/{}", name.trim_start_matches('@').replace("/", "__"))
+}
```

Comment:
There is a function `types_package_name`. Maybe we should consolidate these two?